metrics-operator/controllers/common/providers/dynatrace/client/config.go

```diff
@@ -10,6 +11,10 @@ type apiConfig struct {
 	authURL          string
 	oAuthCredentials oAuthCredentials
 }
+type secretValues struct {
```

Comment:
Let's not duplicate the struct and rather use the already created one here https://github.com/keptn/lifecycle-toolkit/pull/2713/files#diff-8c95f809867d2d5c4602696e58742e661e29b56732c142f93554b060ef7f4d23R28